Chiazokam/Politico
# #163498718 create the create user endpoint

#### What does this PR do?
This PR tests and creates the create user endpoint

#### Description of Task to be completed?
- write tests for every test case
- set up jwt
- set up bcrypt
- separate the server for the data structures
- write validators for the user input
- create the create user endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-create-user-163498718`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/auth/signup `
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
            "firstname":"Dan",
            "lastname": "Mabel",
            "othername": "Cory",
            "email": "daun@mabel.com",
            "phone": "+234-7033567234",
            "passportUrl": "www.passplort.jpeg",
            "password": "lilian"
        }
`
#### What are the relevant pivotal tracker stories?
[#163498718](https://www.pivotaltracker.com/story/show/163498718)

#### Screenshots
![image](https://user-images.githubusercontent.com/26940294/51829771-8aef2500-22ee-11e9-82de-5e902471d103.png)
